package.json

```diff
@@ -2,11 +2,12 @@
   "name": "humblescraper",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "index.ts",
   "scripts": {
     "test": "jest",
     "testw": "jest --watch",
-    "tsc": "tsc"
+    "tsc": "tsc",
+    "start": "npx ts-node index.ts"
```

Comment:
You can now execute the index.ts program with `npm run start`

Reply:
👍